modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -40,6 +40,12 @@ namespace {
     constexpr std::string_view _loggerCat = "SkyBrowserModule";
 
 
+    bool browserBelongsToCurrentNode(std::string identifier) {
+        // Convert the last char to an int
+        int nodeId = static_cast<int>(identifier[identifier.length() - 1] - '0');
```

Comment:
This could be dangerous when the number of nodes is bigger than 9

How about:
```cpp
std::string str;
int nodeId;
auto res = scn::scan("{}{}", str, nodeId);
if (!res) {
  throw ghoul::RuntimeError(...);
}
else {
```

Reply:
I agree that there should be a better solution. Unfortunately, the Identifier of the browser also ends with a number, which could be multiple digits. I suppose I could change the Identifier to something like SkyBrowser0_ and then the node number comes after that? So then I could split it where the _ is